### DISCUSSION TITLE
Sponsoring section does not list all sponsorships

### DISCUSSION THREAD

**Discussion by cesarcoatl** [community/community#30] [Mobile] [answered]
While the main profile counts all sponsorships, the Sponsoring section appears to only list individuals or user accounts.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the feedback here! I've confirmed that this is broken on iOS, and filed an issue for our team to patch this up 😊

**Reply by cesarcoatl** [reply]
Fixed on version 1.6.0.
